pkg/rpc/client/nep11.go

```diff
@@ -0,0 +1,165 @@
+package client
+
+import (
+	"fmt"
+
+	"github.com/nspcc-dev/neo-go/pkg/core/transaction"
+	"github.com/nspcc-dev/neo-go/pkg/encoding/address"
+	"github.com/nspcc-dev/neo-go/pkg/io"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/callflag"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/nspcc-dev/neo-go/pkg/vm/emit"
+	"github.com/nspcc-dev/neo-go/pkg/vm/opcode"
+	"github.com/nspcc-dev/neo-go/pkg/vm/stackitem"
+	"github.com/nspcc-dev/neo-go/pkg/wallet"
+)
+
+// NEP11Decimals invokes `decimals` NEP11 method on a specified contract.
+func (c *Client) NEP11Decimals(tokenHash util.Uint160) (int64, error) {
+	return c.nepDecimals(tokenHash)
+}
+
+// NEP11Symbol invokes `symbol` NEP11 method on a specified contract.
+func (c *Client) NEP11Symbol(tokenHash util.Uint160) (string, error) {
+	return c.nepSymbol(tokenHash)
+}
+
+// NEP11TotalSupply invokes `totalSupply` NEP11 method on a specified contract.
+func (c *Client) NEP11TotalSupply(tokenHash util.Uint160) (int64, error) {
+	return c.nepTotalSupply(tokenHash)
+}
+
+// NEP11BalanceOf invokes `balanceOf` NEP11 method on a specified contract.
+func (c *Client) NEP11BalanceOf(tokenHash, owner util.Uint160) (int64, error) {
+	return c.nepBalanceOf(tokenHash, owner, nil)
+}
+
+// TODO: support NEP11TokensOf? (it returns iterator)
```

Comment:
It'd need some fancy script and paging support. Let's create a separate issue for that.

Reply:
And other iteratorish methods.

Reply:
https://github.com/nspcc-dev/neo-go/issues/1864